src/configs/cmd_duration.rs

```diff
@@ -15,6 +15,7 @@ pub struct CmdDurationConfig<'a> {
     pub disabled: bool,
     pub show_notifications: bool,
     pub min_time_to_notify: i64,
+    pub notify_ignore_commands: Vec<&'a str>,
```

Comment:
This is a simple list of commands to ignore.  I'm not sure if we want to be more flexible for pattern matching (regex? list of regexes? regexes based on the entire commandline (command + args)?).